app/views/journals/index.html.erb

```diff
@@ -1,15 +1,19 @@
 <table class="table table-striped text-center">
```

Comment:
I think you can delete this template at all

Reply:
index.html.erb?